docs/manual/installation/contao-installtool.de.md

```diff
@@ -8,6 +8,10 @@ aliases:
 weight: 90
 ---
 
+{{% notice info %}}
+Das Contao-Installtool existiert ab Contao 5 nicht mehr.
+{{% /notice %}}
```

Comment:
This is not really helpful in my opinion. You should either explain what to do instead, or link somewhere, where it is explained.